Another_notebook.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    x, y = rng.multivariate_normal(mean, cov, n).T
</pre><p>Change this to <code>np.random.randn(3,3)</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/agamemnonc/reviewnb-test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>